### ISSUE TITLE
File sharing completely broken in Ubuntu 20.04

### ISSUE THREAD

**Issue by BubuOT** [flathub/im.riot.Riot#294]
After https://github.com/flathub/im.riot.Riot/pull/293 file sharing doesn't work anymore on Ubuntu 20.04. The flatpak version of this Ubuntu is too old to support the portal version element requires, resulting in no filesystem access whatsoever.

**Comment by BubuOT** [comment]
This might also be broken for debian stable, which has flatpak 1.10.7
backports has 1.12.7 though, which supposedly should work.

(it surely is also broken on buster and those backports only have 1.10.7)

**Comment by BubuOT** [comment]
Fixed (in the sense that nobody will be updated to a broken version at least) by https://github.com/flathub/im.riot.Riot/pull/295

I'll leave this open for others to find.

**Comment by vbatts** [comment]
now updating this app (or "all flatpaks") is erroring on ubuntu 20.04:

```shell
$ flatpak update -y
Looking for updates…


        ID                   Branch         Op        Remote         Download
 1. [✗] im.riot.Riot         stable         u         flathub        < 114.4 MB

Error: im.riot.Riot needs a later flatpak version
Updates complete.
error: There were one or more errors
```

**Comment by BubuOT** [comment]
@vbatts yes, this is expected. As far as I can see you have one of the following options:

* Update to 22.04
* upgrade flatpak from the flatpak ppa
* switch to element from the element ppa

**Comment by vbatts** [comment]
yep. I can not update to newer ubuntu, and did not find a flatpak backport ppa, so I'm in progress of using their https://element.io/get-started#linux-details apt repo.

**Comment by vbatts** [comment]
search engines didn't pull up https://flatpak.org/setup/Ubuntu but there it is. 
Updating to flatpak `1.14.0-2flatpak1~20.04.1` now...

**Comment by vbatts** [comment]
(and this works nicely)
